server/manager/src/main/java/org/apache/accumulo/manager/Manager.java

```diff
@@ -1616,14 +1625,16 @@ private TServer setupReplication()
         Property.MANAGER_REPLICATION_COORDINATOR_THREADCHECK, maxMessageSizeProperty);
 
     log.info("Started replication coordinator service at " + replAddress.address);
+    // TODO KEVIN RATHBUN this thread creation exists within a task which is labeled non-critical
+    // so assuming these are as well.
     // Start the daemon to scan the replication table and make units of work
-    replicationWorkThread = Threads.createThread("Replication Driver",
+    replicationWorkThread = Threads.createNonCriticalThread("Replication Driver",
         new org.apache.accumulo.manager.replication.ReplicationDriver(this));
     replicationWorkThread.start();
 
     // Start the daemon to assign work to tservers to replicate to our peers
     var wd = new org.apache.accumulo.manager.replication.WorkDriver(this);
-    replicationAssignerThread = Threads.createThread(wd.getName(), wd);
+    replicationAssignerThread = Threads.createNonCriticalThread(wd.getName(), wd);
```

Comment:
this thread creation exists within a task which is labeled non-critical so assuming these are as well.

Reply:
> within a task which is labeled non-critical

Where is it labeled non-critical?